src/DiabloInterface.Server/Handlers/ItemRequestHandler.cs

```diff
@@ -46,7 +46,7 @@ public static List<ItemInfo> GetItemsByLocations(D2DataReader dataReader, List<B
             return Items;
         }
 
-        private ItemInfo(
+        public ItemInfo(
```

Comment:
probably better to not use stuff from Server in Business part of the code, but it is already mixed before this change, so np . i ll clean this at some point ^^